packages/render-ui/src/preview-prompt.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import * as color from './color';
+
+export class PreviewPrompt extends Component {
+    static propTypes = {
+        classes: PropTypes.object,
+        prompt: PropTypes.string,
+        tagName : PropTypes.string,
+        className : PropTypes.string
+    }
+
+    render() {
+        const { prompt, classes, tagName, className} = this.props;
+        const CustomTag = (tagName) ? tagName : "div";
+        const customClasses = `${(className) ? classes[className] : ""} ${classes.promptTable}`
```

Comment:
put the default at the front and use an OR: ``${classes.promptTable} ${classes[className] || ''}``